### PR TITLE
Added additional resources section to readthedocs

### PR DESCRIPTION
**Pull request by engelmi**
Added additional resources section to readthedocs

### CODE REVIEW

File: doc/docs/resources.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Additional Resources
+
+- [Blog Post about a lightweight service manager](https://www.redhat.com/en/blog/hirte-renamed-eclipse-bluechi)
```
Comment (mwperina) [inline doc/docs/resources.md]: Could you please add also a link to DevConf 2023 presentation?

https://www.youtube.com/watch?v=8RiRiviSVqM

Comment (mwperina) [inline doc/docs/resources.md]: Just wondering, wouldn't it be better to structure link names? Something like:

```
<Presentation or blog post name>, <presenter(s)>, <Name of the conference or blog>, <date>
```

Comment (engelmi) [reply doc/docs/resources.md]: I was using the format from [systemd](https://systemd.io/) (under `Videos for Users and Administrators` sections). Both is fine for me. What do you think?

Comment (engelmi) [reply doc/docs/resources.md]: Done :heavy_check_mark:

Comment (mwperina) [reply doc/docs/resources.md]: Hmm, to be frank I don't like that almost every line starts with "Presentation about" or "Blog post about", the name of the presentation/blog should be descriptive/interesting enough ...

Comment (engelmi) [reply doc/docs/resources.md]: Changed.
@mwperina PTAL

**Review by mwperina — APPROVED** [review]
Looks much more readable for me, thanks

**Comment by engelmi** [comment]
The testing-farm job seems to be stuck. Since this PR only changes the readthedocs its not required. 
Lets merge.

**Pull request merged**